examples/gno.land/p/demo/helper/helper.gno

```diff
@@ -0,0 +1,10 @@
+package helper
```

Comment:
the current package is too small to justify taking this generic package name.

Reply:
do you have some recommend for this ?

Reply:
removed Helper package, alternatively i defined helper function within package